src/sentry/auth/authenticators/u2f.py

```diff
@@ -49,10 +56,30 @@ def is_available(cls):
         url_prefix = options.get("system.url-prefix")
         return url_prefix and url_prefix.startswith("https://")
 
+    def _create_credential_object(self, registeredKey):
+        return base.AttestedCredentialData.from_ctap1(
+            websafe_decode(registeredKey["keyHandle"]),
+            websafe_decode(registeredKey["publicKey"]),
+        )
+
     def generate_new_config(self):
         return {}
 
-    def start_enrollment(self):
+    def start_enrollment(self, user, is_webauthn_register_ff):
+        if is_webauthn_register_ff:
```

Comment:
let's change this to `has_webauthn_register`. I think we don't really use `ff` to mean feature flag.